message.go

```diff
@@ -7,10 +7,12 @@ import (
 
 // Message is a data structure representing kafka messages.
 type Message struct {
-	Offset int64
-	Key    []byte
-	Value  []byte
-	Time   time.Time
+	Topic     string
+	Partition int32
```

Comment:
The package uses int for partition numbers everwhere, let’s keep it consistent here.

Reply:
Sure.  I should have looked further.  I saw Conn declared partition int32 and followed.